server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -31,6 +31,8 @@ parsers.exactLength=Exact length of %s must be %s bytes
 parsers.intRangeValue =Allowed range is %s to %s
 parsers.alphaNumeric=Only alphanumerics are allowed for %s
 parsers.startsWith=String must starts with %s values
+parsers.stringLengthRange=String length must be between %s and %s
+parsers.allowedStringValues=Only allowed value(s) %s
```

Comment:
How is about adding a colon here? Like this:
```
Only allowed value(s): %s
```

Reply:
this ^

Reply:
@temanbrcom Added colon. @zeibura Are you expecting anything else?

Reply:
Nope that's fine

Reply:
Ok. thanks